project/CodeCoverageSettings.scala

```diff
@@ -24,7 +24,7 @@ object CodeCoverageSettings {
 
   val settings: Seq[Setting[_]] = Seq(
     ScoverageKeys.coverageExcludedPackages := excludedPackages.mkString(";"),
-    ScoverageKeys.coverageMinimumStmtTotal := 77,
+    ScoverageKeys.coverageMinimumStmtTotal := 76,
```

Comment:
After removing unnecessary repositories we have:
[info] Statement coverage.: 78.16%
https://build.tax.service.gov.uk/job/VOA_TCTR/job/tenure-cost-and-trade-records-frontend-pr-builder/201/console

So no reason to decrease coverage from 77% to 76%

Reply:
Sorted. Ive changed it back to 77% and created a method in Session to use 👍